website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,114 @@
+## Usage
+
+### When to use
+
+- To navigate to a new destination, either internally or externally.
+- As a standalone element or within a Button Set, use a **(Link) Standalone**
+
+### When not to use
+
+- Within a body of text, use a **Link (Inline)**
+- To trigger an action or event, like a form submission, use a [Button](/components/button/overview)
+
+### Size
+
+<Hds::Link::Standalone @size="large" @text="Large" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />
```

Comment:
```suggestion

<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />

```
These need some breathing room, visually.

Reply:
Agreed, but I kinda punted on that for this first pass. I'll commit this to see if it helps.